docs/getting-started/datapackage-find-prepare-share-guide.md

```diff
@@ -6,26 +6,28 @@ authors: Dmytro Gierman, Branko Djordjevic
 
 **In this guide, we describe a way to gather information and make it presentable and easy to use for your customers.**
```

Comment:
This line should not be there right ...  I think your summary is something like

In this guide we walk through creating a data package from scratch and publishing to the DataHub. As a bonus, we'll also show you how to collect data and clean up before packing it for publication.